### DISCUSSION TITLE
How can i collect ISO image with custom docker images!

### DISCUSSION THREAD

**Discussion by umlumpa** [Security-Onion-Solutions/securityonion#12108] [General]
Hi guys! appreciate for this product! i want to collect ISO image with custom docker images how can i do it! why ISO images i have problem with internet and with ISO image i can solve this problem! Help me please!

**Comment by Acewiza** [comment]
Not exactly SO-related, but assuming you're trying to image an installed system:  https://clonezilla.org/
